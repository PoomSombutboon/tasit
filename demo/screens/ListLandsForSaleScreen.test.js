import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import { ListLandsForSaleScreen } from "./ListLandsForSaleScreen";

describe("ListLandsForSaleScreen", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    const landsForSale = [];
    const setLandsForSale = () => {};
    const selectLandToBuy = () => {};
    expect(
      shallow(
        <ListLandsForSaleScreen
          landsForSale={landsForSale}
          setLandsForSale={setLandsForSale}
          selectLandToBuy={selectLandToBuy}
        />
      )
    ).toMatchSnapshot();
  });
});
