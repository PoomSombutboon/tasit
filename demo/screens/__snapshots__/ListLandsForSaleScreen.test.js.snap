// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListLandsForSaleScreen renders the component 1`] = `
<LandsForSaleList
  landsForSale={Array []}
  renderItem={[Function]}
/>
`;
